src/cascadia/TerminalApp/App.cpp

```diff
@@ -26,6 +26,7 @@ namespace winrt::TerminalApp::implementation
         }
 
         Initialize();
+        HighContrastAdjustment(::winrt::Windows::UI::Xaml::ApplicationHighContrastAdjustment::None);

```

Comment:
A comment leaving a bit of explanation and the effects of this above the call would be appreciated. (As well as perhaps a reference to what it influences in the XAML files for future cross-referencing when we add more UI or update what we have.) Someone *could* go look up this PR with `git blame` and the docs page on `HighContrastAdjustment`, but I feel like it's a thing that should be explained in-line.